From 3a52f76694effb6748050625abb8d4f79ce82acb Mon Sep 17 00:00:00 2001
From: Nate Drude <nate.d@variscite.com>
Date: Fri, 10 Sep 2021 10:59:28 -0500
Subject: [PATCH 10/11] imx8mn-var-som: Move CONFIG_CSF_SIZE to
 imx8mn_var_som_defconfig

This fixes the following error when CONFIG_IMX_HAB=y

    Authenticate image from DDR location 0x401fcd40...
    Error: Invalid IVT structure

Signed-off-by: Nate Drude <nate.d@variscite.com>
---
 configs/imx8mn_var_som_defconfig | 1 +
 include/configs/imx8mn_var_som.h | 4 ----
 2 files changed, 1 insertion(+), 4 deletions(-)

diff --git a/configs/imx8mn_var_som_defconfig b/configs/imx8mn_var_som_defconfig
index dfdb9712ff..d0005a558b 100644
--- a/configs/imx8mn_var_som_defconfig
+++ b/configs/imx8mn_var_som_defconfig
@@ -11,6 +11,7 @@ CONFIG_SPL_MMC_SUPPORT=y
 CONFIG_SPL_SERIAL_SUPPORT=y
 CONFIG_NR_DRAM_BANKS=1
 CONFIG_SPL=y
+CONFIG_CSF_SIZE=0x2000
 CONFIG_SPL_IMX_ROMAPI_LOADADDR=0x48000000
 CONFIG_SPL_TEXT_BASE=0x912000
 CONFIG_FIT=y
diff --git a/include/configs/imx8mn_var_som.h b/include/configs/imx8mn_var_som.h
index 5c94987024..360e1be568 100644
--- a/include/configs/imx8mn_var_som.h
+++ b/include/configs/imx8mn_var_som.h
@@ -13,10 +13,6 @@
 
 #include "imx_env.h"
 
-#ifdef CONFIG_SECURE_BOOT
-#define CONFIG_CSF_SIZE			0x2000 /* 8K region */
-#endif
-
 #define CONFIG_SPL_MAX_SIZE		(148 * 1024)
 #define CONFIG_SYS_MONITOR_LEN		(512 * 1024)
 #define CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_USE_SECTOR
-- 
2.35.1

